session1/examples/generics.ts

```diff
@@ -0,0 +1,33 @@
+//Generic type variables
+function checkPasswordLength<T>(password: T) {
```

Comment:
1. Let's please add the line of code where we are invoking this function to show the actual output.
2. Why are using ES5 function syntax here but not the ES6 arrow function?

Reply:
Done

Reply:
TBD